modules/installation-dns-user-infra.adoc

```diff
@@ -149,12 +149,12 @@ api-int.ocp4.example.com.	IN	A	192.168.1.5 <2>
 ;
 bootstrap.ocp4.example.com.	IN	A	192.168.1.96 <4>
 ;
-master0.ocp4.example.com.	IN	A	192.168.1.97 <5>
-master1.ocp4.example.com.	IN	A	192.168.1.98 <5>
-master2.ocp4.example.com.	IN	A	192.168.1.99 <5>
+control-plane-0.ocp4.example.com.	IN	A	192.168.1.97 <5>
```

Comment:
There seems to be an inconsistency, the labels referenced with <control_plane><n> whereas the example shows control_plane-0, it should be control_plane0.